common/app/views/fragments/metaData.scala.html

```diff
@@ -113,3 +113,11 @@
 }
 
 <meta name="google-site-verification" content="LR-FN6c2gIEUoo3k049w1nxyHykmac5ZE3SaUOiKc30" />
+
+@* Metadata to deep-link into native apps from facebook. http://applinks.org/documentation/ *@
+<meta property="al:ios:url" content="theguardian://">
```

Comment:
The url property defines just a custom schema that must match the one defined in the App. The url of the content is contained in a JSON. The final url will be:  
<code>[url]?al_applink_data=JSON_ENCODED_DATA</code> 

Said that I think that I should change the url to be something like  _theguardian://applinks_